packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard/signup-wizard.tsx

```diff
@@ -54,17 +63,27 @@ const SignupWizard = ({ closeWizard }: TSignupWizardProps) => {
                         secondary_button_label={localize('Back')}
                         onChangeStep={onChangeStep}
                     >
-                        <Wizard.Step title='Step 1' is_fullwidth>
+                        <Wizard.Step title='Country of issue' is_fullwidth is_submit_disabled={!is_country_selected}>
```

Comment:
@hamid-deriv could we extract this step in separate component for better readablity?

Reply:
That's a good idea. Changed.

Reply:
We have the `CountrySelector` component for this step. I think having another component to wrap this component is not a good idea.

Reply:
@hamid-deriv we can create `CountryStep` or smth else. It will be much cleaner if we will have every step as a separate component IMHO
@farzin-deriv @bahar @vinu what do you think?

Reply:
Updated :+1:

Reply:
@hamid-deriv thanks!